src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -12,6 +12,26 @@ The resulting ZIP archive is composed of:
 - a folder for each network containing
   + TBD
 
+## Schema Validation
+To validate the produced XML document, you can use `xmllint` tool 
+
+```shell
+apt install libxml2-utils
```

Comment:
you may precise that this command works on ubuntu/debian

Reply:
I added some precisions in 34f2c52e9aa1a9fb984a613c086e3846752248ef.